pkg/cmd/log.go

```diff
@@ -0,0 +1,223 @@
+// Copyright KubeArchive Authors
+// SPDX-License-Identifier: Apache-2.0
+package cmd
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
+	"net/http"
+	"net/url"
+	"strings"
+
+	"github.com/spf13/cobra"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/cli-runtime/pkg/genericclioptions"
+	"k8s.io/client-go/rest"
+)
+
+type LogOptions struct {
+	ContainerName string
+	Name          string
+	Resource      string
+	GroupVersion  string
```

Comment:
In `kubectl logs` the expected way of providing the resource and the groupversion is in the same argument.

E.g:

`kubectl logs deployments/my-deploy`
` kubectl logs -n kubearchive deployments.apps/kubearchive-operator`

should we consider moving to the same format?